instrumentation/logback/logback-mdc-1.0/library/README.md

```diff
@@ -32,6 +32,12 @@ dependencies {
 
 ### Usage
 
+If the `otel.instrumentation.logback-mdc.add-baggage` system property (or the
+`OTEL_INSTRUMENTATION_LOGBACK_MDC_ADD_BAGGAGE` environment variable) is set to `true`,
+key/value pairs in [baggage](https://opentelemetry.io/docs/concepts/signals/baggage/) will be added to the context too.
+
+- `baggage.<entry_name>`
+
```

Comment:
in library instrumentation logback appender is configured via logback.xml not system properties or environment variables, see https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/b8c28df05b815114420812c3909c5b3ebf84f2f5/instrumentation/logback/logback-mdc-1.0/library/src/addBaggageTest/resources/logback.xml#L23